src/main/resources/db/migration/V0001__Base_version.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE public.case_payment_orders (
+                         id integer not null PRIMARY KEY,
+                         created_timestamp timestamp without time zone not null,
+                         effective_from timestamp without time zone not null,
+                         case_id BIGINT not null,
+                         case_type_id character varying(70),
+                         action character varying(70) not null,
+                         responsible_party character varying(1024) not null,
+                         order_reference character varying(70) not null,
+                         created_by character varying(70)
```

Comment:
I think this should always be provided too?
```suggestion
                         created_by character varying(70) not null
```

Reply:
Yes, fair enough I guess - will update data model in confluence as well

Reply:
Probably the same for `case_type_id` too, as that's a mandatory parameter for the create API

Reply:
Fixed in commit https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/f47a41148bdbc4123ae3b8761b453b6e8b535f8d